datasets/mlsum/mlsum.py

```diff
@@ -0,0 +1,139 @@
+from __future__ import absolute_import, division, print_function
+
+import json
+import os
+
+import nlp
+
+
+_CITATION = """\
+@article{scialom2020mlsum,
+  title={MLSUM: The Multilingual Summarization Corpus},
+  author={Scialom, Thomas and Dray, Paul-Alexis and Lamprier, Sylvain and Piwowarski, Benjamin and Staiano, Jacopo},
+  journal={arXiv preprint arXiv:2004.14900},
+  year={2020}
+}
+"""
+
+_DESCRIPTION = """\
+We present MLSUM, the first large-scale MultiLingual SUMmarization dataset. 
+Obtained from online newspapers, it contains 1.5M+ article/summary pairs in five different languages -- namely, French, German, Spanish, Russian, Turkish. 
+Together with English newspapers from the popular CNN/Daily mail dataset, the collected data form a large scale multilingual dataset which can enable new research directions for the text summarization community. 
+We report cross-lingual comparative analyses based on state-of-the-art systems. 
+These highlight existing biases which motivate the use of a multi-lingual dataset. 
+"""
+_URL = "https://gitlab.lip6.fr/scialom/mlsum_data/-/raw/master/MLSUM/"
+_LANG = ["de", "es", "fr", "ru", "tu"]
+
+
+class MlsumConfig(nlp.BuilderConfig):
+    def __init__(self, data_url, **kwargs):
+        """BuilderConfig for MLQA
```

Comment:
`MLQA` to `MLSUM` ?

Reply:
Indeed I used that script as a base, I corrected it